src/components/layout/nav.js

```diff
@@ -767,6 +767,7 @@ export const NavPartners = ({ no_login_signup }) => {
                                     is_affiliate_sign_in_link
                                     target="_blank"
                                     primary
+                                    style={{ width: '12rem' }}
                                 >
                                     <span>{localize('Affiliate & IB log in')}</span>
```

Comment:
I found a bug , if the page is on top (when sign up button not showing ), the login button can't be clicked

I guess theres a problem with the 

`pointer-events`  and `cursor` css styling 

from `NavRight` on `line 181` 
> might need to differentiate the styling for `<a>` between `sign in` `and sign up`

---
does this fix is only for `english` page , or need to adjust for other language as well

Reply:
Alright, thanks for spotting that. Will get it fixed 👍

Reply:
It breaks down for certain languages that are much longer, for e.g. Polish. How do you guys usually cater for all the other languages?

Reply:
usually we make 1 single fix for all language 
most of the time, the english - which usually have the shorter sentence, will look longer 
if it's unachievable, then it will be better to discuss it with the design team